logger/main.go

```diff
@@ -2,102 +2,106 @@ package main
 
 import (
 	"flag"
-	"fmt"
 	"log"
 	"os"
 	"os/signal"
 	"strconv"
-	"strings"
 	"syscall"
 	"time"
 
-	"github.com/coreos/go-etcd/etcd"
-	"github.com/deis/deis/logger/syslogd"
+	"github.com/deis/deis/logger/configurer"
+	"github.com/deis/deis/logger/publisher"
+	"github.com/deis/deis/logger/storage"
+	"github.com/deis/deis/logger/syslogish"
+	"github.com/deis/deis/logger/weblog"
 )
 
 var (
 	logAddr         string
+	logHost         string
 	logPort         int
-	drainURI        string
 	enablePublish   bool
-	publishHost     string
-	publishPath     string
-	publishPort     string
+	webAddr         string
+	webPort         int
+	etcdHost        string
+	etcdPort        string
+	etcdPath        string
+	configInterval  int
 	publishInterval int
 	publishTTL      int
 )
 
 func init() {
 	flag.StringVar(&logAddr, "log-addr", "0.0.0.0", "bind address for the logger")
+	flag.StringVar(&logHost, "log-host", getopt("HOST", "127.0.0.1"),
```

Comment:
Taking a hint from the old implementation and from other Deis components and trying to avoid external dependencies wherever I can.  (The etcd-go library is the only external dependency for this component.)